js/ui/grid_core/ui.grid_core.column_chooser.js

```diff
@@ -206,16 +171,45 @@ const columnChooserMembers = {
         } else {
             this._popupContainer.option(dxPopupOptions);
         }
+
+        this.setPopupAttributes();
+    },
+
+    setPopupAttributes: function() {
+        const isSelectMode = this.isSelectMode();
+
+        this._popupContainer.setAria({
+            role: 'dialog',
+            label: messageLocalization.format('dxDataGrid-columnChooserTitle')
+        });
+
+        this._popupContainer.$wrapper()
+            .toggleClass(this.addWidgetPrefix(COLUMN_CHOOSER_DRAG_CLASS), !isSelectMode)
+            .toggleClass(this.addWidgetPrefix(COLUMN_CHOOSER_SELECT_CLASS), isSelectMode);
+
+        this._popupContainer.$content().addClass(this.addWidgetPrefix(COLUMN_CHOOSER_LIST_CLASS));
+
+        if(isSelectMode && !this._columnsController.isBandColumnsUsed()) {
+            this._popupContainer.$content().addClass(this.addWidgetPrefix(COLUMN_CHOOSER_PLAIN_CLASS));
+        }
```

Comment:
Minor comment.

It's not good practice to use function call inside the if's condition
Anyway we should know the result of the ```isBandColumnsUsed```
Let's move it to a separate variable on the top of this function:

```suggestion
    setPopupAttributes: function() {
        const isSelectMode = this.isSelectMode();
        const isBandColumnUsed = this._columnsController.isBandColumnsUsed();

        this._popupContainer.setAria({
            role: 'dialog',
            label: messageLocalization.format('dxDataGrid-columnChooserTitle')
        });

        this._popupContainer.$wrapper()
            .toggleClass(this.addWidgetPrefix(COLUMN_CHOOSER_DRAG_CLASS), !isSelectMode)
            .toggleClass(this.addWidgetPrefix(COLUMN_CHOOSER_SELECT_CLASS), isSelectMode);

        this._popupContainer.$content().addClass(this.addWidgetPrefix(COLUMN_CHOOSER_LIST_CLASS));

        if(isSelectMode && !isBandColumnUsed) {
            this._popupContainer.$content().addClass(this.addWidgetPrefix(COLUMN_CHOOSER_PLAIN_CLASS));
        }
```